src/Gui/Selection.cpp

```diff
@@ -715,10 +717,18 @@ bool SelectionSingleton::addSelection(const char* pDocName, const char* pObjectN
         Notify(Chng);
         signalSelectionChanged(Chng);
 
+        snprintf(buf,512,"Selected: %s.%s.%s (%f,%f,%f)",pDocName
+                                                        ,pObjectName
+                                                        ,pSubName
+                                                        ,x,y,z);
+
 #ifdef FC_DEBUG
-        Base::Console().Log("Sel : Add Selection \"%s.%s.%s(%f,%f,%f)\"\n",pDocName,pObjectName,pSubName,x,y,z);
+        Base::Console().Log(buf);

```

Comment:
Yeah and further thinking about it i am not sure if unified approach (status bar/log) is the best approach from consistency point of view.  Therefore i will revert the original log message and that is that from my side for now.